apps/storybook.namekit.io/stories/Namekit/utils.ts

```diff
@@ -8,32 +8,37 @@ import {
   buildENSName,
   now,
   subtractSeconds,
+  getDomainName,
+  buildNFTRefFromENSName,
+  ENSName,
+  MAINNET,
 } from "@namehash/ens-utils";
 import { Normalization } from "@namehash/nameguard";
 
-export const REGISTRATION_OF_EXPIRING_SOON_DOMAIN = {
+export const REGISTRATION_OF_EXPIRING_SOON_DOMAIN: Readonly<Registration> = {
   registrationTimestamp: subtractSeconds(now(), buildDuration(1000n)),
   expirationTimestamp: addSeconds(now(), buildDuration(4000n)),
   expiryTimestamp: null,
   primaryStatus: PrimaryRegistrationStatus.Active,
   secondaryStatus: SecondaryRegistrationStatus.ExpiringSoon,
 };
 
-export const REGISTRATION_OF_EXPIRED_DOMAIN = {
+export const REGISTRATION_OF_EXPIRED_DOMAIN: Readonly<Registration> = {
   registrationTimestamp: subtractSeconds(now(), buildDuration(1000n)),
   expirationTimestamp: addSeconds(now(), buildDuration(4000n)),
   expiryTimestamp: null,
   primaryStatus: PrimaryRegistrationStatus.Expired,
   secondaryStatus: SecondaryRegistrationStatus.GracePeriod,
 };
 
-export const REGISTRATION_OF_NORMAL_EXPIRATION_STATUS = {
-  registrationTimestamp: null,
-  expirationTimestamp: null,
-  expiryTimestamp: null,
-  primaryStatus: PrimaryRegistrationStatus.Active,
-  secondaryStatus: null,
-};
+export const REGISTRATION_OF_NORMAL_EXPIRATION_STATUS: Readonly<Registration> =
```

Comment:
I don't understand several things here:

1. Why is this named "...NORMAL..."?  "Normal" doesn't have any special meaning in this context. Is "Active" the word you're looking for?
2. Why are we hard-coding each of the `Registration` objects in this file? Why aren't we using one of our utility functions to build a `Registration` object? There can be some special exceptions to this, but in general it should be a red-flag if we are manually building objects like these.
3. The `Registration` object being built here isn't valid. Invariants are being violated that shouldn't be. Ex: If it is `PrimaryRegistrationStatus.Active` then `registrationTimestamp` must not be `null`, etc... This problem would be completely avoided if point 2 was followed correctly. This is why we should in general avoid manually building objects this way. The probability that you're going to do it wrong is too high. It's also difficult to maintain.
4. It looks like the logic to build a `Registration` object hasn't been added to `ens-utils` yet? Is there any issue to add that?

Please review and update the way that each `Registration` created in this file is defined.

Reply:
Hi @lightwalker-eth !

You are right, in order to better make these concepts available:

1. `getDomainRegistration` and `getRegistrationForActiveDomain` functions are now being exported from `ens-utils`
2. `registration` and `parsedName` params were renamed to `domainRegistration` and `domainName`
3. `DomainName` typed params were changed to receive `ENSName`s
4. `DomainCard` typed params were removed and replaced by specific params that are typed as `Address`, `Registration` and `ENSName`, so no dependency on `DomainCard` exists